src/go/k8s/tests/e2e-v2/upgrade-values-check/02-upgrade-remove-listeners.yaml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
NIT: The file name src/go/k8s/tests/e2e-v2/upgrade-values-check/02-upgrade-remove-listeners.yaml could indicate that stroage definition was removed too.

Reply:
I decided not to approach the test this way. I may add a new step later.